spec/rails_helper.rb

```diff
@@ -12,6 +12,9 @@
 # Add additional requires below this line. Rails is not loaded until this point!
 
 Capybara.javascript_driver = :poltergeist
+Capybara.register_driver :poltergeist do |app|
+  Capybara::Poltergeist::Driver.new(app, timeout: 1.minute)
```

Comment:
Are there specific tests that need this timeout? Might be good to just put the timeout on the tests that need it rather than everywhere. That way you know which ones are long running.